packages/db/tests/test-providers.js

```diff
@@ -1,31 +0,0 @@
-/* eslint-disable unicorn/no-null */
-const test = require('ava');
-
-const {
-  nullifyUndefinedProviderValues,
-} = require('../dist/provider');
-
-test('nullifyUndefinedProviderValues sets undefined provider values to "null"', async (t) => {
```

Comment:
Major nit territory: but I prefer this separation of test files. This function isn't defined in the same file as the model (intentionally), so I prefer to keep the tests separate as well

Reply:
I'm not sure I follow - This was moved to translate/test-providers to be consistent with the other tests (e.g. the rest of the tests in the tests/translate directory.

Reply:
Hmm, not a nit.    f25733c0b